logger.go

```diff
@@ -55,6 +56,39 @@ func (l Level) String() string {
 	}
 }
 
+// List of predefined log Formats
+const (
+	Json Format = iota
+	Logfmt
+)
+
+// Format represents the predefined log format.
+type Format int
+
+// formatFromString returns a formatter instance appropriate for the given format name.
+func FormatFromString(format string) (Formatter, error) {
+	switch format {
+	case "json":
+		return JSONFormat(), nil
+	case "logfmt":
+		return LogfmtFormat(), nil
+	default:
+		return nil, xerrors.Errorf("log: invalid log format: %s", format)
+	}
+}
+
+// String returns the string representation of the level.
+func (f Format) String() string {
+	switch f {
+	case Json:
```

Comment:
Not sure if this should not be JSON instead of Json

Reply:
Going with the Go convention, it should be JSON I guess. Will fix it.